lib/inc/pxp-agent/util/bolt_helpers.hpp

```diff
@@ -4,11 +4,38 @@
 #include <leatherman/locale/locale.hpp>
 #include <leatherman/curl/client.hpp>
 #include <leatherman/json_container/json_container.hpp>
+#include <pxp-agent/util/bolt_module.hpp>
 
 #include <tuple>
 
 namespace PXPAgent {
 namespace Util {
+
+  // Hard-code interpreters on Windows. On non-Windows, we still rely on permissions and #!
+  const std::map<std::string, std::function<std::pair<std::string, std::vector<std::string>>(std::string)>> BUILTIN_INTERPRETERS {
+  #ifdef _WIN32
+      {".rb",  [](std::string filename) { return std::pair<std::string, std::vector<std::string>> {
+          "ruby", { filename }
+      }; }},
+      {".pp",  [](std::string filename) { return std::pair<std::string, std::vector<std::string>> {
+          "puppet", { "apply", filename }
+      }; }},
+      {".ps1", [](std::string filename) { return std::pair<std::string, std::vector<std::string>> {
+          "powershell",
+          { "-NoProfile", "-NonInteractive", "-NoLogo", "-ExecutionPolicy", "Bypass", "-File", filename }
+      }; }}
```

Comment:
Are these just the set from Bolt?

Reply:
They appear to be yeah: https://github.com/puppetlabs/bolt/blob/master/lib/bolt/transport/powershell.rb#L8

they look right to me fwiw